CSS/style.css

```diff
@@ -42,6 +42,81 @@ body {
   border-top-left-radius: 16px;
 }
 
+/* ------------------------menu start----------------------------------- */
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 84vh;
+  margin: 0 auto;
+  padding: 1.6rem;
+  background-color: #6070ff;
+  z-index: 4;
+  display: none;
+  flex-direction: column;
+  align-items: start;
+  justify-content: start;
+  gap: 2rem;
+  border-radius: 5%;
+  opacity: 90%;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: start;
+  padding: 3px;
+  margin-left: 12px;
+}
+
+.mobile-menu ul {
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.list-of-course li {
+  width: 75px;
+  height: 24px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px;
+  margin-right: 0.2rem;
+  text-align: center;
+}
+
+.mobile-menu ul li {
+  margin-bottom: 1rem;
+}
+
+.list-of-course a {
+  text-decoration: none;
+  color: #396df2;
+  margin-left: -3px;
+}
+
+.mobile-menu ul li a {
+  font-size: 2rem;
+  font-weight: 500;
+  line-height: 2.5rem;
+  color: var(--dark-black);
+  text-decoration: none;
+  margin: 1rem;
+  transition: ease-in-out 2s;
+}
+
+.mobile-menu-close-btn {
+  align-self: end;
+}
+
+/* ---------------------------------------------menu end----------------------------- */
```

Comment:
- [ ] Your mobile menu creation is commendable :+1:. However, please ensure that it matches the Figma template as it is a requirement for the project. Specifically, the mobile menu should occupy the entire height of the page, and the links should have a white text color and be positioned more towards the left. It is also important to check the paddings and margins to ensure that they align with the Figma template. This will help ensure consistency and accuracy in the project.

**Your design**

![Screenshot from 2023-04-18 05-08-19](https://user-images.githubusercontent.com/105117832/232746106-d39b73ff-54c9-4dbe-ad52-08d592f84a48.png)

**Figma**

![Screenshot from 2023-04-18 05-09-51](https://user-images.githubusercontent.com/105117832/232746157-e9a8e54b-2f7d-445e-bb56-d0a6157199b1.png)